s4e-backend/src/main/java/pl/cyfronet/s4e/config/TimeConfig.java

```diff
@@ -1,14 +1,23 @@
 package pl.cyfronet.s4e.config;
 
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
 
 import javax.annotation.PostConstruct;
+import java.time.Clock;
 import java.util.TimeZone;
 
 @Configuration
-public class TimeZoneConfig {
+public class TimeConfig {
     @PostConstruct
     void started() {
         TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
     }
+
+    @Bean
+    @Profile({ "!test", "integration" })
```

Comment:
its "not test"

Reply:
I use this condition not to load this bean in non-integration tests. Otherwise, I wouldn't be able to define an alternative `clock` bean in test scope, as is done in TestClockConfiguration.